plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -709,6 +709,29 @@ private void testCorruptedTableLocation(String tableName, Path tableLocation, bo
         }
     }
 
+    /**
+     * @see deltalake.stats_with_minmax_nulls
+     */
+    @Test
+    public void testStatsWithMinMaxValuesAsNulls()
+            throws Exception
+    {
+        String tableName = "stats_with_minmax_nulls" + randomNameSuffix();
+        Path tableLocation = Files.createTempFile(tableName, null);
+        copyDirectoryContents(new File(Resources.getResource("deltalake/stats_with_minmax_nulls").toURI()).toPath(), tableLocation);
+        getQueryRunner().execute(
+                format("CALL system.register_table('%s', '%s', '%s')", getSession().getSchema().orElseThrow(), tableName, tableLocation));
+
+        assertThat(query("SELECT * FROM " + tableName))
+        .matches("""
+                    VALUES
+                    (0, 1),
+                    (1, 2),
+                    (3, 4),
+                    (NULL, NULL)
+                    """);
```

Comment:
Can we verify `SHOW STATS FOR` statement as well? I believe this statement also failed without this PR.